docs/config/README.md

```diff
@@ -1380,3 +1381,28 @@ files = ["foo"]       # can specify filters
 when = """ test "$HOME" == "$PWD" """
 prefix = " transcending "
 ```
+
+## PureScript
+
+The `purescript` module shows the currently installed version of PureScript version.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `spago.dhall` file
+- The current directory contains a \*.purs files
+
+### Options
+
+| Variable   | Default        | Description                                                  |
+| ---------- | -------------- | ------------------------------------------------------------ |
+| `symbol`   | `"<=> "`       | The symbol used before displaying the version of PureScript. |
+| `style`    | `"bold black"` | The style for the module.                                    |
```

Comment:
The default here is `bold black` but in the config it is `bold white` should this be changed?